osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -351,7 +351,7 @@ private void select(CarouselItem item)
         /// <summary>
         /// Half the height of the visible content.
         /// <remarks>
-        /// This is different from the height of <see cref="ScrollContainer{T}.displayableContent"/>, since
+        /// This is different from the height of <see cref="ScrollContainer{T}"/>.displayableContent, since
```

Comment:
Referring to private member will surprise user more when sources of framework and game are separated.